src/core/plugins/download-url.js

```diff
@@ -8,9 +8,21 @@ export default function downloadUrlPlugin (toolbox) {
   let { fn } = toolbox
 
   const actions = {
-    download: (url)=> ({ errActions, specSelectors, specActions, getConfigs }) => {
+    download: (url)=> ({ errActions, specSelectors, specActions, getConfigs, authSelectors }) => {
       let { fetch } = fn
       const config = getConfigs()
+      let headers = {
+        "Accept": "application/json,*/*"
+      }
+      if ( config.sendAuthHeaderOnJsonFetch ) {
+        let authorizedMethods = authSelectors.authorized().map( (val, key) => {
+          return { val: val, key: key }
+        }).toArray()
+        let apiKeyMethods = authorizedMethods.filter(i => i.key === "api_key")
+        if ( apiKeyMethods.length ) {
+          headers["Authorization"] = apiKeyMethods[0].val.get("value")
```

Comment:
`"Authorization"` is a magic string, that can be configured in the security schemes. Also, the `0` seems quite arbitrary. Why not the last?

Reply:
in what file should it be defined? also, there is an "api_key" magic string - the name of the auth method, should I put it in the default settings index.js? 

It does not make sense to have > 1 authorizations with API key, because we cannot tell which one should we use to send an auth header with, so I guess we could either just grab the first authorized. It would be overkill to notify a user that he has multiple auth methods and he needs to choose one of them.